### PR TITLE
Move docs on Optimism Contract Mapping to yaml

### PR DESCRIPTION
**Pull request by augustog**
This is a very small change to keep our documentation consolidated on YAML files, should be treated as low priority.

This way we can have spell-specific documentation (like in the case of optimism contracts) living as a github-friendly README.md file, *and* as part of dbt-docs.

image

What I did:
- [x] add docs block to yml file
- [x] add docs block marker to readme file
- [x] validate `dbt docs generate` locally

### CODE REVIEW

**Comment by augustog** [comment]
@team not at all a high priority here, but I'd like y'all's take of having this as our way of keeping spell-specific docs in the future

**Review by soispoke — APPROVED** [review]

**Comment by soispoke** [comment]
lgtm

File: dbt_project.yml [hunk 1]
```diff
@@ -15,7 +15,6 @@ test-paths: ["tests"]
 seed-paths: ["seeds"]
 macro-paths: ["macros"]
 snapshot-paths: ["snapshots"]
-docs-paths: ["docs"]
```
Comment (davidkell) [inline dbt_project.yml:18]: AFAIK if you remove this, we need to move the `docs/overview.md` file into the `models` directory. This file is used to render the custom welcome page in the docs site

**Review by jeff-dude — APPROVED** [review]
i haven't dug too deep into the docs side yet, but in theory this lgtm. i think it's certainly an area we can experiment with

File: dbt_project.yml (hunk 1 above)
Comment (augustog) [reply dbt_project.yml:18]: let me test it locally to ensure it still works!

**Review by augustog — COMMENTED** [review]
Latest commit addresses @davidkell 's point. (indeed the config change reverted the docs to the default).

**Comment by jeff-dude** [comment]
@augustog please let me know if i need to monitor anything execution-wise of this code in prod. otherwise, it's merged and should be set 👍

**Pull request merged**